.github/ISSUE_TEMPLATE/simple-atomic-parity.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Simple/Atomic Feature Parity
-about: Did you find something inconsistent or unexpected between the two platforms? 
+about: Did you find something inconsistent or unexpected between the two platforms?
 title: ''
-labels: "Simple/Atomic Parity"
+labels: Simple/Atomic Parity
```

Comment:
I have no idea why this has pushed itself (I've created the template via the UI). I think it's OK to leave it tho :)